src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -0,0 +1,671 @@
+<?php
+/**
+ * Interactivity API: WP_Interactivity_API class.
+ *
+ * @package WordPress
+ * @subpackage Interactivity API
+ */
+
+/**
+ * Class used to process the Interactivity API in the server.
```

Comment:
```suggestion
 * Class used to process the Interactivity API on the server.
```

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/5953/commits/e8daa1c89140249202d9976ef73d5eef9bd78c2c.